.github/workflows/coverage-move-only.yaml

```diff
@@ -0,0 +1,55 @@
+name: "Test Coverage For Move"
+on:
+  # Trigger if any of the conditions
+  #   Any changes to the directory `third_party/move/`
+  push:
+    paths:
+      - 'third_party/move/**'
```

Comment:
just fyi, this won't trigger for PRs that created from forks. Only for PRs that are created from within a branch of aptos-core.

Reply:
thanks for the reminder, @geekflyer!

Reply:
I am wondering if a PR is created from a fork, how should I specify the path? @geekflyer